src/main/webapp/app/course/plagiarism-cases/instructor-view/detail-view/plagiarism-case-instructor-detail-view.component.ts

```diff
@@ -60,6 +60,9 @@ export class PlagiarismCaseInstructorDetailViewComponent implements OnInit, OnDe
      * and saves the point deduction in percent
      */
     savePointDeductionVerdict(): void {
+        if (!this.isStudentNotified()) {
+            return;
+        }
```

Comment:
Should we maybe throw an error or send an issue to Sentry in that case?

Reply:
Alright, I'll throw an error 👍

Reply:
I added the errors here: https://github.com/ls1intum/Artemis/pull/5543/commits/7c83f80470057208f4521546cb3c39a1e4cf6e9d